hw2/04-background.html

```diff
@@ -1,29 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>04 - Background</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="04-background.js"></script>
-  </head>
-  <body>
-    <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
-      </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
-      </div>
-      <div>
-        <input />
-      </div>
-    </main>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>04 - Background</title>
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+		/>
+		<script defer src="04-background.js"></script>
+	</head>
+	<body class="d-flex justify-content-center align-content-center p-5">
+		<main class="bg-light w-25 p-15px rounded">
+			<h3 class="m-3" style="white-space: nowrap">Exercise 04 - Background</h3>
+			<p class="m-3">
+				The background color will change every few seconds (as indicated below).
+				You can stop this behavior and you can also change how often the
+				background color changes.
+			</p>
+			<div>
+				<label class="m-3 mb-0">Background change interval (seconds): </label>
+				<input class="m-3 mt-0" type="number" min="3" id="interval" />
+			</div>
+			<div>
+				<button type="button" class="btn btn-primary m-3">Start</button>
```

Comment:
Since you start with the interval going this should start as a stop button

Reply:
corrected the logic.